RecoTracker/TkSeedGenerator/python/GlobalSeedsFromPairsWithVertices_cff.py

```diff
@@ -21,7 +21,7 @@
       maxElement = cms.uint32(1000000)
     ),
     RegionFactoryPSet = cms.PSet(
-      RegionPSetWithVerticesBlock,
+      RegionPSet = globalTrackingRegionWithVertices.RegionPSet,
```

Comment:
Please clone the `globalTrackingRegionWithVertices.RegionPSet`. If anyone modifies `globalSeedsFromPairsWithVertices.RegionPSet` the changes get propagated to `globalTrackingRegionWithVertices.RegionPSet` as well.